### DISCUSSION TITLE
Just for fun...

### DISCUSSION THREAD

**Discussion by ferdnyc** [BiglySoftware/BiglyBT#2909] [Welcome]
My current graph of all-time BiglyBT transfer activity, spanning just over 10 years (which, technically, is 4 years longer than BiglyBT _per se_ has even **existed**). Weekly upload and download fluctuations totaling nearly `15 TB` up and nearly `10 TB` down.

![image](https://user-images.githubusercontent.com/538020/224454914-ba361b53-ef61-44ba-9c18-cc7352b55f0f.png)

(And no, I have no idea how I managed to download over 100 GB in a single week, not just once but _twice_ during the mid-2010s. Or how I managed to upload nearly 90GB one week in mid-2016. My upload bandwidth was absolute crap until very recently. I finally upgraded from cable broadband to fiber service less than a month ago.)

**Comment by parg** [comment]
90GB in a week is 150k/sec, no way you could achieve that?
